.ci/atime/tests.R

```diff
@@ -130,9 +130,7 @@ test.list <- list(
   N = 10^seq(1, 7),
   setup = quote({ 
     L <- replicate(N, 1, simplify = FALSE)
-    data.table:::setattr(L, "class", NULL)    
-    data.table:::setDT(L)  
   }),
-  expr = quote(data.table:::setDT(DT)),
+  expr = quote(data.table:::setattr(L, "class", NULL); data.table:::setDT(DT)),
```

Comment:
please use {} to break over two lines instead of using ;
also it should be L not DT